kingpin/actors/aws/ecs.py

```diff
@@ -1044,11 +1044,30 @@ def _update_service(self, service_name, existing_service, override=None):
         if override is not None:
             update_parameters.update(override)
 
+        
+        
         self.log.info('Updating service.')
-
-        yield self.api_call(
-            self.ecs_conn.update_service,
-            **update_parameters)
+        try:
+            self.log.info('Checking if existing service is active.')
+            service_is_active = (existing_service and
```

Comment:
Checking if service being inactive is really the state its in before the deployment fails (it should be, based on AWS docs): https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_UpdateService.html